### PR TITLE
Add LastPass policies

### PR DESCRIPTION
**Pull request by joshfreemanIO**

### CODE REVIEW

File: lastpass/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# LastPass
+
+## Why Do We Use LastPass?
+
+A password's strength is depends upon entropy. The more possibilities your password can be, the stronger your it becomes. The strongest passwords are random passwords; unfortunately, humans are terrible at memorizing random data, leading us choose easily rememberable and easily crackable passwords.
```
Comment (laurenfackler) [inline lastpass/README.md]: I think you want "dependent" here.

Comment (laurenfackler) [inline lastpass/README.md]: "stronger your it becomes" :)

File: lastpass/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The second threat to passwords is password reuse. This isn't necessarily a problem with the password as it is threats exposed by services having their data compromised. If a password is reused, any service using that password is also compromised. Reusing passwords or generating passwords that are unique only by the site-derived passwords like **strongpasswordgoogle** expose your accounts to takeover.
+
+LastPass allows us to generate and store unique and cryptographically strong passwords for each site and application we use.
+
+## Choosing a Master Password
+
+Your Master Password is the password you use to access all the usernames, passwords, and other secrets you have stored in LastPass. It is literally the key to your kingdom. Using family member's names, important dates, song lyrics, and other simple passwords are too weak for any level of safety.
+
+We need a random password but are unable to remember highly complex passwords like. **i&%ndTLf7Hyk**? Let have [XKCD explain Diceware](https://xkcd.com/936/).
```
Comment (laurenfackler) [inline lastpass/README.md]: "Let's have" vs "Let have"

File: lastpass/README.md (hunk 1 above)
Comment (joshfreemanIO) [inline lastpass/README.md]: Yoda

**Pull request merged**